example/android/settings.gradle

```diff
@@ -3,4 +3,4 @@ rootProject.name = 'GoogleSigninSampleApp'
 include ':app'
 
 include ':react-native-google-signin'
-project(':react-native-google-signin').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-google-signin/android')
+project(':react-native-google-signin').projectDir = new File(rootProject.projectDir, '../../android')
```

Comment:
shouldn't the example app maintain the way things would be done normally so the users can reference it?

Reply:
yeah, that is a good question. The way I do the development is that I open up the example project and than use it to develop the native module, that is in the left pane in android studio, the `react-native-google-signin` project is pointing to the code of this repo, not to node_modules, so if I make changes in the module, they ready to be committed to git.
What workflow do you use? cc @jozan